### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by thegcat** [aristocratos/bpytop#131]
**Describe the bug**

`bpytop` crashes when cycling through the themes.

**To Reproduce**

[Steps to reproduce the behavior:]

1. Start `bpytop`
2. Go to the menu and cycle through the themes.

[A clear and concise description of what you expected to happen.]

Theme change instead of crash 😄 

[If applicable, add screenshots to help explain your problem.]

**Info (please complete the following information):**
 - bpytop version: 1.0.35
 - psutil version: 5.7.2
 - (Linux) Linux distribution and version: Debian 10.5
 - Terminal used: ssh + iTerm
 - Font used: FiraCode Nerd Font
 - Python version: 3.7.3

**Additional context**

contents of `~/.config/bpytop/error.log`

```
21/09/20 (12:26:37) | ERROR: Data collection thread failed with exception: list index out of range
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/bpytop.py", line 2559, in _runner
    collector._draw()
  File "/usr/local/lib/python3.7/dist-packages/bpytop.py", line 3438, in _draw
    ProcBox._draw_fg()
  File "/usr/local/lib/python3.7/dist-packages/bpytop.py", line 2436, in _draw_fg
    vals += [f'{THEME.gradient["proc_color" if val < 100 else "process"][val if val < 100 else val - 100]}']
IndexError: list index out of range
21/09/20 (12:26:37) | WARNING: Exiting with errorcode (1). Runtime 0:00:32
```

**Comment by aristocratos** [comment]
@thegcat 
Possible fix added in v1.0.36

If it's still crashing please answer the following:
1. Do you have any custom themes or only the default unmodified themes present?
2. Does it crash every time you try to change theme? If not, does it crash on any specific theme?
3. What's the system specs for the machine running bpytop?

**Comment by thegcat** [comment]
Stock themes and it did it reliably 3 times in a row after getting through less than 8 themes each time.

Couldn't reproduce it now, so seems to be fixed, thanks!
